lib/authenforce/enforcers.go

```diff
@@ -1,32 +1,27 @@
 package authenforce
 
 import (
+	"context"
+
 	"github.com/cuvva/cuvva-public-go/lib/cher"
-	"github.com/wearemojo/mojo-public-go/lib/authparsing"
+	"github.com/wearemojo/mojo-public-go/lib/merr"
 )
 
-type Enforcer func(*authparsing.AuthState) error
-
-func UnsafeNoAuthentication(*authparsing.AuthState) error {
-	return nil
-}
+type (
+	Enforcer  func(context.Context, any, map[string]any) error
+	Enforcers []Enforcer
```

Comment:
Often packages provide a function to build its built-in types, e.g. new/wrap - may be worth consideration